src/styled/pages.ts

```diff
@@ -0,0 +1,13 @@
+import Styled from '@emotion/styled';
+import { colorMap } from './theme';
+
+const PageWrapper = Styled('div')`
+    background: white;
```

Comment:
Try not to use named colors but rather hex values. If we continue to use `white` and `black` often might be best to store `#000` and `#fff` in `const`.